ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1138,7 +1138,7 @@ Rectangle {
             Shortcut {
                 enabled: messageInputField.activeFocus
                 sequence: "Ctrl+Meta+Space"
-                onActivated: emojiBtn.clicked()
+                onActivated: emojiBtn.clicked({})
```

Comment:
@borismelnik since the parameter should be `MouseEvent`, maybe more appropriate value here is `null` instead of `{}`?

Reply:
`{}` is empty object which is not `null`

Reply:
And what's different? I fixed it